### ISSUE TITLE
Google Vision doesn't seem to work with gs:// URLs

### ISSUE THREAD

**Issue by erikaxel** [GoogleCloudPlatform/google-cloud-ruby#880]
Hi, when doing this in my console:
```
vision = Google::Cloud.new('my-project', ENV['GOOGLE_AUTH_FILE']).vision
image = vision.image 'gs://my-project.appspot.com/myimage.jpg'
```
I get the following error:
```NameError: undefined local variable or method `url' for #<Google::Cloud::Vision::Image:0x007fa378e33348>```

It comes from the to_s method in image. 
If I try to use a method directly to avoid the to_s, such as:
`text = vision.image('gs://my-project.appspot.com/my-image').text`

I get the following error
`Google::Cloud::InvalidArgumentError: image-annotator::Malformed request.: Invalid GCS path 
specified: `

In the last case, it seems like the URL is not set.

Additional info:
- The project works with local files, so the setup should be ok
- I see that there is not acceptance test for gs-files, only local path files, which leads me to believe that this is a real issue

**Comment by blowmage** [comment]
Hi @erikaxel, thanks so much for opening this issue! I identified the problem and submitted a patch for it. I hope we will have this reviewed, merged, and released today.

Thanks again for making us aware of this!

**Comment by quartzmo** [comment]
@erikaxel Thank you for reporting this. The fix is on the way.

**Comment by erikaxel** [comment]
That was quick! Thank you very much. I can verify that it now works. (I checked out source locally and tested it)

**Comment by blowmage** [comment]
This is resolved in [google-cloud-vision v0.20.2](https://rubygems.org/gems/google-cloud-vision/versions/0.20.2). Thanks!
